bench/bench_init.py

```diff
@@ -35,8 +35,8 @@ def run(self):
 
     def get_cifar_data(self):
         train_x, train_y = cifar_utils.filter_data(*cifar_utils.load_cifar(
-            self.cifar_dir_path, cifar_filename="cifar_train.data",
-            norm=False))
+            self.cifar_dir_path, cifar_filename="cifar_train.data", norm=False)
```

Comment:
Revert

Reply:
This is needed to fix formatting issues we've been having